adb_graphics/figures/maps.py

```diff
@@ -406,48 +408,48 @@ def _title(self):
             not_labeled.extend([h.short_name for h in self.hatch_fields])
             if not any(list(set(cf.short_name).intersection(['pres']))):
                 title = cf.vspec.get('title', cf.field.long_name)
-                contoured.append(f'{title} ({cf.units}, hatched)')
+                contoured.append(f'{title} ({cf.units}, hatched)\n')
 
         # Add descriptor string for the important contoured fields
         if self.contour_fields:
             for cf in self.contour_fields:
                 if cf.short_name not in not_labeled:
                     title = cf.vspec.get('title', cf.field.long_name)
+                    title = title.replace("Geopotential", "Geop.")
                     contoured.append(f'{title}')
                     contoured_units.append(f'{cf.units}')
 
-        contoured = ', '.join(contoured)
+        contoured = ''.join(contoured)  # The join method makes contoured a string, enabling linefeeds
```

Comment:
Did you check out an example of the title on a plot that has multiple contours and hatches?